Makefile

```diff
@@ -14,9 +15,14 @@ requirements:
 static:
 	python scripts/compile_sass.py
 
-validate_accessibilty:
+validate_quality:
+	pep8 google_maps/; \
+	pylint google_maps/; \
+	npm run lint
```

Comment:
Without it, if one of the tests fails (i.e. pep8), the ones after that one don't run. 
That works for travis since we need all of them to pass, but I think it's better to have all 3 tests run regardless, especially when running "make validate" manually.